src/modules/scraper/service/DepositsGapDetectionCron.ts

```diff
@@ -0,0 +1,173 @@
+      }
+      const gapIntervalsWithBlocks = await this.attachBlocksToDepositGaps(gapIntervals, chainId);
+      if (gapIntervalsWithBlocks.length > 0) {
+        this.logger.log(
+          `Detected deposits gap for chainId: ${chainId} (lastDepositId: ${lastDepositId}) ${JSON.stringify(
+            gapIntervalsWithBlocks,
+          )}`,
+        );
+        chainGapIntervals[chainId] = gapIntervalsWithBlocks;
+      }
+    }
+    await this.monitoringService.postSlackMessage(this.formatSlackPayload(chainGapIntervals));
+  }
+
+  private async attachBlocksToDepositGaps(gapIntervals: DepositGapInterval[], chainId: number) {
+    const gapIntervalsWithBlocks: DepositGapIntervalWithBlocks[] = [];
+
+    for (const interval of gapIntervals) {
+      const previousKnownDeposit = await this.depositRepository.findOne({
+        where: { sourceChainId: chainId, depositId: LessThan(interval.fromDepositId) },
+        order: { depositId: "DESC" },
+      });
+      const nextKnownDeposit = await this.depositRepository.findOne({
+        where: { sourceChainId: chainId, depositId: MoreThan(interval.toDepositId) },
+        order: { depositId: "ASC" },
+      });
+
+      gapIntervalsWithBlocks.push({
+        ...interval,
+        previousKnownDeposit: previousKnownDeposit
+          ? {
+              depositId: previousKnownDeposit.depositId,
+              blockNumber: previousKnownDeposit.blockNumber,
+            }
+          : // using null instead of undefined because JSON.stringify will remove undefined values
+            null,
+        nextKnownDeposit: nextKnownDeposit
+          ? {
+              depositId: nextKnownDeposit.depositId,
+              blockNumber: nextKnownDeposit.blockNumber,
+            }
+          : // using null instead of undefined because JSON.stringify will remove undefined values
+            null,
+      });
+    }
+
+    return gapIntervalsWithBlocks;
+  }
+
+  formatSlackPayload(chainGapIntervals: Record<number, DepositGapIntervalWithBlocks[]>) {
+    const text = [];
+
+    for (const chainId of Object.keys(chainGapIntervals)) {
+      const gapIntervals: DepositGapIntervalWithBlocks[] = chainGapIntervals[chainId];
+      text.push({
+        type: "rich_text_section",
+        elements: [
+          {
+            type: "text",
+            text: `\n\nChainId ${chainId}`,
+            style: {
+              bold: true,
+            },
+          },
+        ],
+      });
+      const bullets = [];
+      for (const gapInterval of gapIntervals) {
+        bullets.push({
+          type: "rich_text_section",
+          elements: [
+            {
+              type: "text",
+              text: `deposits from ${gapInterval.fromDepositId} to ${gapInterval.toDepositId}\n(previous: ${gapInterval.previousKnownDeposit?.depositId} ${gapInterval.previousKnownDeposit?.blockNumber}, next: ${gapInterval.nextKnownDeposit?.depositId} ${gapInterval.nextKnownDeposit?.blockNumber}`,
```

Comment:
NIT: Maybe we can add more labels in the text body? It wasn't clear to me which number is the deposit id and which the block number by only looking at the slack message

Reply:
done